src/libraries/System.Text.Json/tests/Common/ReferenceHandlerTests/ReferenceHandlerTests.IgnoreCycles.cs

```diff
@@ -456,21 +456,11 @@ private async Task Test_Serialize_And_SerializeAsync<T>(T obj, string expected,
 
             if (objType != typeof(object))
             {
-                json = await JsonSerializerWrapperForString.SerializeWrapper(obj, options);
-                Assert.Equal(expected, json);
-
-                using var ms1 = new MemoryStream();
-                await JsonSerializerWrapperForStream.SerializeWrapper(ms1, obj, options).ConfigureAwait(false);
-                json = Encoding.UTF8.GetString(ms1.ToArray());
+                json = await Serializer.SerializeWrapper(obj, options);
```

Comment:
So these can be combined now since this test class will run under both string+stream?